### PR TITLE
Added Function sample to trigger AML pipeline

### LINKED ISSUE #42: Create a sample to trigger Azure ML pipelines from Azure Functions
1. Document on why and what we want to demonstrate in this sample. 
2. Create unit tests for the sample.
3. Create CI/CD pipelines for the sample.

### PR DESCRIPTION
**Pull request by aykhara**
Added new sample which demonstrate how to trigger Azure ML pipeline without access to ADO and AML by using Azure Functions.

### CODE REVIEW

**Review by h2floh — DISMISSED** [review]
Great work I have nothing to add, very carefully written README carving out the use case for this.

File: common/pipeline_trigger/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+When using Azure Machine Learning(AML) Service, there are cases where it is necessary to operate the pipeline under various business scenario. This sample demonstrates a scenario where users without access to AML can trigger an AML pipeline using an azure function.
+
+**What does this sample demonstrate:**
+- How to invoke AML pipeline using Azure Function
+
+**What doesn't this sample demonstrate**:
+- Azure Machine Learning Service Creation.
+
+![architecture](media/architecture.png)
```
Comment (angie4u) [inline common/pipeline_trigger/README.md:12]: If I check this readme with preview, it seems like image is broken! 
image

File: common/pipeline_trigger/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+- [Linting and Testing](##Linting-and-Testing)
+
+## 1. Create Azure Resources
+Please refer to this [README.md](environment_setup\provisioning\README.md) under environment_setup/provisioning
+
+
+## 2. Set AML access right to Azure Function
+
+1. Enable System Assigned Identity
+  ![system_assigned_identity.png](media/system_assigned_identity.png)
```
Comment (angie4u) [inline common/pipeline_trigger/README.md:72]: same here! I think all images are currently broken!

File: common/pipeline_trigger/README.md (hunk 1 above)
Comment (h2floh) [reply common/pipeline_trigger/README.md:12]: If you look at it in the branch it seems to work https://github.com/liupeirong/MLOpsManufacturing/tree/aykhara/pipelinetrigger/common/pipeline_trigger

**Review by h2floh — APPROVED** [review]

File: common/pipeline_trigger/README.md (hunk 2 above)
Comment (aykhara) [reply common/pipeline_trigger/README.md:72]: Thanks @angie4u and @h2floh for reviewing this PR. 
Could you have a look at README on the branch? It seems not broken.
https://github.com/liupeirong/MLOpsManufacturing/tree/aykhara/pipelinetrigger/common/pipeline_trigger

**Review by daisukei777 — APPROVED** [review]
Looks good.

File: common/pipeline_trigger/README.md (hunk 2 above)
Comment (h2floh) [reply common/pipeline_trigger/README.md:72]: I can confirm that it's not broken on the branch

Comment (daisukei777) [reply common/pipeline_trigger/README.md:72]: I confirmed it too.

**Pull request merged**